### ISSUE TITLE
Error of interaction of different cheats. Maybe this information will help in the development of radarhack!!!

### ISSUE THREAD

**Issue by Shuvi-Moro** [danielkrupinski/Osiris#4186]
In short the problem is this, if I run Osiris with MidNight there are 2 possible outcomes:

1. I launch Osiris first. In this case MidNight has a radarhack problem.

2. MidNight launches first. In this case, when you turn on the display of steps in Osiris, the game crashes.

And yes I know it doesn't have to work together, but I think it might help to solve the radarhack problem in the future during its development.

**Comment by hamadzidani** [comment]
> In short the problem is this, if I run Osiris with MidNight there are 2 possible outcomes:
> 
> 1. I launch Osiris first. In this case MidNight has a radarhack problem.
> 2. MidNight launches first. In this case, when you turn on the display of steps in Osiris, the game crashes.
> 
> And yes I know it doesn't have to work together, but I think it might help to solve the radarhack problem in the future during its development.

is there a radarhack feature?

**Comment by Shuvi-Moro** [comment]
> > In short the problem is this, if I run Osiris with MidNight there are 2 possible outcomes:
> > 
> > 1. I launch Osiris first. In this case MidNight has a radarhack problem.
> > 2. MidNight launches first. In this case, when you turn on the display of steps in Osiris, the game crashes.
> > 
> > And yes I know it doesn't have to work together, but I think it might help to solve the radarhack problem in the future during its development.
> 
> is radarhack feature already implemented?

Yes, but the cheat is paid, but there is very well done radarhack, sees from the very spavnoy. I would like to see something like that in Osiris.

**Comment by angelaly11** [comment]
oh man, had the exact same issue last month when trying to use radarhack with another tool and it drove me crazy too. anyway, @hamadzidani asked if the radarhack feature is already implemented in Osiris, but since you mentioned it's paid and there's a good one out there (@PEXCH-SENTXMENT), i'm guessing not yet?

tbh, i think @PEXCH-SENTXMENT's suggestion is actually pretty cool - having that kind of radarhack feature would be super helpful. maybe the dev can look into integrating it in future updates? anyway, if you're still struggling with these issues, check out this thread where i found some good solutions: https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 (there's a fix for similar crashes in that post). hope it helps!
